### PR TITLE
Implement cmd of github repository in order to move quickly

### PR DESCRIPTION
**Pull request by shopetan**
## What
- Add cmd of github repository in order to move quickly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -353,10 +353,11 @@ $ biko github [product] [flag(s)]
...
 
-| Product   | What                  | Command     | Flags(Optional) |
-| :----:    | :----:                | :----:      | :----:          |
-| Dashboard | Open github Dashboard | `dashboard`, `db` | `--org`         |
-| Trending  | Open github Treinding | `trending`, `t`   | `--language, -l`, `--since, -s` |
+| Product    | What                   | Command     | Flags(Optional) |
+| :----:     | :----:                 | :----:      | :----:          |
+| Dashboard  | Open github Dashboard  | `dashboard`, `db` | `--org`         |
+| Trending   | Open github Treinding  | `trending`, `t`   | `--language, -l`, `--since, -s` |
+| Repository | Open github Repository | `repository`, `r` | `--org`, `--name, -n` |
```
Comment (shopetan) [inline README.md]: set format

File: cli/github.go [hunk 2]
```diff
@@ -86,3 +87,27 @@ func newGithubTrendingCmd() cli.Command {
...
+
+func newGithubRepositoryCmd() cli.Command {
+	return cli.Command{
+		Name:    "repository",
+		Aliases: []string{"r"},
+		Usage:   "Open the repository page",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "org",
+				Usage: "Organization to open",
```
Comment (shopetan) [inline cli/github.go]: `org` flag include some `github_user_id` like `KeisukeYamashita` . 🤕 
So please comment if you have any idea.

Comment (KeisukeYamashita) [reply cli/github.go]: Let's take a look at what Github API defines the API.

For listing repos they have two APIs.

```
 /users/:username/repos
```

```
/orgs/:org/repos
```

So, from this point of view, I suggest you to implement two flags `-org` and `-user`.
If the user specifies two flags in a single command, we can return an error.

Any thoughts?

Comment (shopetan) [reply cli/github.go]: @KeisukeYamashita 
I just add flag of`users` !

File: providers/github/github.go [hunk 3]
```diff
@@ -51,26 +51,44 @@ func (p *Provider) GetTargetURL() (string, error) {
...
 			p.join(language)
 			return
 		}
+	case "repository":
+
+		users := p.Ctx.String("users")
+		org := p.Ctx.String("org")
+		if users != "" && org != "" {
+			p.URL = p.baseURL
+			return
```
Comment (shopetan) [inline providers/github/github.go]: You told me that we can return an `error` but `addProductPath` has not any type of variable.
So I changed that if both `users` and `org` was filled in, return base url.
I think we need discuss it if change interface: +1:

Comment (KeisukeYamashita) [reply providers/github/github.go]: > I think we need discuss it if change interface: +1:

Ahhhh, I see. Thank you for catching this. I agree with you.

File: providers/github/github.go [hunk 4]
```diff
@@ -51,26 +51,44 @@ func (p *Provider) GetTargetURL() (string, error) {
...
+		} else if users != "" {
```
Comment (KeisukeYamashita) [inline providers/github/github.go]: We can remove the `else if` because there are early returns.

**Pull request merged**